packages/api/src/router/review.ts

```diff
@@ -72,7 +73,38 @@ export const reviewRouter = {
 
   create: protectedProcedure
     .input(CreateReviewSchema)
-    .mutation(({ ctx, input }) => {
+    .mutation(async ({ ctx, input }) => {
+      console.log("new review");
```

Comment:
I feel like we can remove the logging since the errors / success return value imply the same message